test/vtctld_web_test.py

```diff
@@ -94,7 +94,11 @@ def _get_keyspaces(self):
 
   def _get_keyspace_element(self, keyspace_name):
     """Get a specific keyspace element given a keyspace name."""
-    return self.driver.find_element_by_id('%s-card' % keyspace_name)
+    element_id = '%s-card' % keyspace_name
+    wait = WebDriverWait(self.driver, 5)
```

Comment:
Can you please move the 5 into a constant? This way the code is also going to be more readable.